flake.nix

```diff
@@ -0,0 +1,28 @@
+{
+  inputs = {
+    cargo2nix.url = "github:cargo2nix/cargo2nix/release-0.11.0";
+    flake-utils.follows = "cargo2nix/flake-utils";
+    nixpkgs.follows = "cargo2nix/nixpkgs";
+  };
+
+  outputs = inputs: with inputs;
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = import nixpkgs {
+          inherit system;
+          overlays = [cargo2nix.overlays.default];
+        };
+
+        rustPkgs = pkgs.rustBuilder.makePackageSet {
+          rustVersion = "1.70.0";
```

Comment:
Here, I might need to update the rust version to 1.75.0 as well.

I think it's better to leave this into a fork and probably update the README, to point to it for Nix users.